app/controllers/api/comments_controller.rb

```diff
@@ -0,0 +1,12 @@
+class Api::CommentsController < ApplicationController
+  def create
+    values = params.require(:comment).permit(:text)
+    comment = Comment.new(text: values[:text], author_id: current_user.id, post_id: params[:post_id])
+
+    if comment.save
+      render json: comment, status: :created
+    else
+      render json: comment.errors, status: :unprocessable_entity
```

Comment:
- [ ] Hello and nice job on the API but it seems that the comment is not rendered as JSON after adding a comment.

![Capture d’écran 2022-08-27 143725](https://user-images.githubusercontent.com/61562626/187032662-dcbe2728-77d1-42f6-a597-0eacb54bf125.png)

![Capture d’écran 2022-08-27 142920](https://user-images.githubusercontent.com/61562626/187032666-90472145-3795-46af-97e5-09fed62b09cd.png)

Reply:
We checked our project and it's work like suppost to be, please be kindly, make the bunlde install and configure your DB, We tested it and works.

Thanks for the project changes.

![image](https://user-images.githubusercontent.com/98621809/187035123-6925f646-a9fb-41b0-9b51-58828a1315ad.png)

Reply:
Hello, before we do the review we are supposed to install everything, as you see it's not rendering the user or the post but the as they are added they also should be in API version like an array of objects.

Reply:
Hello @dasileker !

you mean the response needs to be like this?
![image](https://user-images.githubusercontent.com/99830261/187036843-46bca9d3-8899-41e8-bb4a-530cfd6dd117.png)

This is what it returns whenever a post is made via API, you can see the JSON request and the JSON response.

Reply:
The routes to the api verification are:

localhost:3000/api/users/{[user.id](http://user.id/)}/posts/{[post.id](http://post.id/)}/comments/?api_token=*token generated by the app*

localhost:3000/api/users/{[user.id](http://user.id/)}/posts/{[post.id](http://post.id/)}/?api_token=*token generated by the app*

Reply:
> * [x]  Hello and nice job on the API but it seems that the comment is not rendered as JSON after adding a comment.
> 
> ![Capture d’écran 2022-08-27 143725](https://user-images.githubusercontent.com/61562626/187032662-dcbe2728-77d1-42f6-a597-0eacb54bf125.png)
> 
> ![Capture d’écran 2022-08-27 142920](https://user-images.githubusercontent.com/61562626/187032666-90472145-3795-46af-97e5-09fed62b09cd.png)

I'm just noticing that you are trying to visit a page that renders the comments only, which does not exist (this is why you are getting the no route error). 
to watch the post comments, you need to go to '//localhost:3000/users/1/posts/1' (the Post Show route).

![image](https://user-images.githubusercontent.com/99830261/187039049-d1fc9853-f52b-44fb-9e2b-3621fdd06514.png)

Reply:
Orry for the late reply, Thats what I'm talking about the comments also should be rendred,  but since now it can be viewed in the json please submit for review

Reply:
We need that you kindly reject the last petition for the review to make another submit, because without that we can't do it.

thanks for your review.

![image](https://user-images.githubusercontent.com/98621809/187039572-08cd82b9-92ec-4edf-85a1-e16211fda31d.png)

Reply:
or, you know, aproove it 🤣

Reply:
No need the next reviewer will review it and if everything is good it will be approved